src/aluno/CadastroAlunoPage.tsx

```
@@ -7,37 +7,33 @@ import { Input } from '@/components/ui/input'
+  const [curso, setCurso] = useState<Curso[]>([])
+  const validatedEmail =
+    location.state?.validatedEmail || localStorage.getItem('validatedEmail');
 
-  //Teste para email institucional
   useEffect(() => {
-    if (process.env.NODE_ENV === 'development') {
-      localStorage.setItem('validatedEmail', 'aluno.teste4@alunos.utfpr.edu.br')
-    }
+    CursoService.listar()
+      .then(setCurso)
+      .catch(() => toast.error("Erro ao carregar cursos disponíveis."))
```

ESLint

[error] Replace `"Erro·ao·carregar·cursos·disponíveis."` with `'Erro·ao·carregar·cursos·disponíveis.'`

(prettier/prettier)